config/config.exs

```diff
@@ -53,6 +53,11 @@ config :logger, :console,
 # Use Jason for JSON parsing in Phoenix
 config :phoenix, :json_library, Jason
 
+# Configure Guardian for JWT authentication
+config :bokken, BokkenWeb.Auth.BotManager,
+  issuer: "bokken",
+  secret_key: System.get_env("SECRET_KEY")
```

Comment:
This config should go to `runtime.exs`, otherwise it will have forever the value that had when first time compiling the project. It should be dynamic and read once but at runtime :pray:

Reply:
Learn more about Elixir configuration at runtime [here](https://hexdocs.pm/elixir/1.14.3/Config.html#module-config-runtime-exs).